EIPS/eip-5202.md

```diff
@@ -48,7 +52,32 @@ A blueprint contract MAY insert any bytes (data or code) between the version byt
 
```

Comment:
note - EIP 3540 is referenced as a hypothetical, not as a dependency of this EIP, so this check probably needs to be overridden by a reviewer